charts/agent/values.yaml

```diff
@@ -262,6 +264,7 @@ deployment-cluster-metrics:
 
 ################################################
 daemonset-logs-metrics:
+  enabled: true
```

Comment:
really this should be split into 2 configs, `containers.logs.enabled` and `containers.metrics.enabled`